ksqldb-rest-app/src/main/java/io/confluent/ksql/api/server/QuerySubscriber.java

```diff
@@ -106,7 +106,7 @@ public void handleError(final Throwable t) {
     }
     final KsqlErrorMessage errorResponse = new KsqlErrorMessage(
         ERROR_CODE_SERVER_ERROR, stringBuilder.toString());
-    log.error("Error in processing query {}", stringBuilder);
+    log.error("Error in processing query {}", stringBuilder, t);
```

Comment:
Do you need another `{}` in the string?

Reply:
nope - log4j loggers allow the last argument to be of type `Throwable` and then they just log the entire stack trace gracefully :)